lib/snip.c

```diff
@@ -436,6 +436,20 @@ grn_snip_get_delimiter_regexp(grn_ctx *ctx, grn_obj *snip, size_t *length)
   return snip_->delimiter_pattern;
 }
 
+grn_rc
+grn_snip_expand_cond(grn_ctx *ctx, grn_snip *snip)
+{
+  size_t cond_capacity = snip->cond_capacity * 2;
```

Comment:
How about using more meaningful name?

```suggestion
  size_t new_capacity = snip->cond_capacity * 2;
```

Reply:
Fixed.